test/decl/func/operator.swift

```diff
@@ -422,3 +422,11 @@ func testPostfixOperatorOnTuple<A, B>(a: A, b: B) {
   (§)((a, (b, b), a))
   _ = (a, ((), (b, (a, a), b)§), a)§
 }
+
+func testNonexistentPowerOperationOutOfScope() {
```

Comment:
```suggestion
func testNonexistentPowerOperatorWithPowFunctionOutOfScope() {
```

Something like that, maybe? The name doesn't make sense to me otherwise.